.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [16.x, 18.15.x]
```

Comment:
@kevinpagtakhan your comment says `18.5`. Double checking is this correct?

Reply:
Oops my bad, I mean to say 18.16 and 18.15 respectively